pkg/apis/monitoring/v1/types.go

```diff
@@ -426,6 +432,24 @@ type PrometheusRuleExcludeConfig struct {
 	RuleName string `json:"ruleName"`
 }
 
+// PrometheusObjectReference prometheus object reference
+type PrometheusObjectReference struct {
+	// API version of the referent.
+	APIVersion string `json:"apiVersion,omitempty"`
```

Comment:
not sure if we need the API version?

Reply:
actually this is GroupVersion and for consistency with k8s api maintained the name similar